packages/expo-video/src/VideoPlayer.web.tsx

```diff
@@ -267,27 +273,43 @@ export default class VideoPlayerWeb
     };
 
     video.onratechange = () => {
+      this.emit('playbackRateChange', video.playbackRate, this.playbackRate);
       this._mountedVideos.forEach((mountedVideo) => {
-        if (mountedVideo === video || mountedVideo.playbackRate === video.playbackRate) return;
+        if (mountedVideo.playbackRate === video.playbackRate) return;
```

Comment:
why do we need it? does it endless loop if it's not there?

Reply:
Also, how many events does it emit?

Reply:
You are right, we will have multiple emissions of the same event if a single player is attached to multiple views. It's fixed in the commit below